src/main/java/frc/lib/utils/PathPlannerUtil.java

```diff
@@ -22,29 +22,43 @@
 import edu.wpi.first.wpilibj2.command.Commands;
 import frc.robot.Constants.SwerveConstants;
 import frc.robot.subsystems.Drive;
+import frc.robot.subsystems.Elevator;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Shooter;
 
 public class PathPlannerUtil {
-  private static final NetworkTableInstance kInstance = NetworkTableInstance.getDefault();
-
-  private static final DoubleArraySubscriber kTargetPoseSub = kInstance.getDoubleArrayTopic("/PathPlanner/targetPose")
+  private static final DoubleArraySubscriber kTargetPoseSub = NetworkTableInstance.getDefault()
+      .getDoubleArrayTopic("/PathPlanner/targetPose")
       .subscribe(new double[] { 0.0, 0.0, 0.0 });
 
-  public static void configure(Drive drive) {
+  public static void configure(Drive drive, Intake intake, Shooter shooter, Elevator elevator) {
     HolonomicPathFollowerConfig config = new HolonomicPathFollowerConfig(SwerveConstants.translationalPID,
         SwerveConstants.rotationalPID,
-        SwerveConstants.maxModuleSpeedMPS, SwerveConstants.driveBaseRadiusMeter, new ReplanningConfig(true, true));
+        SwerveConstants.maxModuleVelocityMPS, SwerveConstants.driveBaseRadiusMeter, new ReplanningConfig(true, true));
 
     AutoBuilder.configureHolonomic(
         drive::getPose,
         drive::resetPose,
         drive::getChassisSpeeds,
         drive::driveRobotCentric,
         config,
-        () -> false,
+        () -> DriverStation.getAlliance().get() == DriverStation.Alliance.Red,
```

Comment:
If we run this on robot init, there is a chance there is no team set and an exception will be thrown. We need to make sure that this configure method is called stemming from the `driverStationConnected` callback in Robot.java